zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -95,13 +95,16 @@ trait Sql {
     sealed case class Cons[A, B <: ColumnSet](head: Column[A], tail: B) extends ColumnSet { self =>
       type ColumnsRepr[T]            = (Expr[Features.Source, T, A], tail.ColumnsRepr[T])
       type Append[That <: ColumnSet] = Cons[A, tail.Append[That]]
+      type TableType
```

Comment:
This can be deleted!

Reply:
@unclebob418 You don't need this `type TableType` anymore, right? So we can delete and then merge

Reply:
fixed & merged changes from master - should be good to go now!